Movie Search/Common/Constants.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Constants.swift
+//  Movie Search
+//
+//  Created by Alexey Grabik on 15/04/2020.
+//
+
+import UIKit
+
+struct Constants {
+    static let searchAPIURLString = "https://api.themoviedb.org/3/search/movie"
```

Comment:
Add comments to explain what each of these constants is used for.
Also do they need to be available for the entire module? It seems like they could be used privately where they are needed instead.